src/features/admin/routes/items-table.tsx

```diff
@@ -0,0 +1,101 @@
+import {Box, Button, IconButton} from "@mui/material";
+import {DataGrid, GridColDef} from "@mui/x-data-grid";
+import React from "react";
+import EditIcon from "@mui/icons-material/Edit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import TabPanel from "./tab-panel";
+import {useDeleteItemMutation} from "../hooks/use-delete-admin-mutation";
+import {queryClient} from "../../../lib";
+import {Link as RouterLink} from "react-router-dom";
+import {Item} from "../../order/types/item";
+
+
+type ItemProps = {
+    items: Item[];
+    value: number;
+};
+
+function ItemsTable({items, value}: ItemProps) {
+
+    const {mutate} = useDeleteItemMutation({
+        onSuccess: () => {
+            queryClient.invalidateQueries(["table-data"]);
+        },
+    });
+
+    function deleteItem(e, row) {  {/*TODO - unused event*/}
```

Comment:
Maybe we can resolve this TODO right now :)

Reply:
is removed